test/NuGet.Core.Tests/NuGet.Configuration.Test/PackageSourceProviderTests.cs

```diff
@@ -2253,7 +2253,7 @@ private void AssertCredentials(PackageSourceCredential actual, string source, st
 
         private static void CreateSettingsFileInTestingGlobalDirectory(TestDirectory directory)
         {
-            var settingsFile = new FileInfo(Path.Combine(directory.Path, "TestingGlobalPath", "NuGet.Config"));
+            var settingsFile = new FileInfo(Path.Combine(directory.Path, "TestingGlobalPath", "NuGet.config"));
```

Comment:
Why does it need to change? It is defined as `NuGet.Config`:

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs#L24-L24

Reply:
It was part of the revert commit. Good catch! I should change it back.